### PR TITLE
docs(Form): add usage example with Checkbox

### PR DESCRIPTION
**Pull request by lucivpav**
Resolves PR: #1808
There are similar examples under _Types_ section. Should I move them under _Usage_ instead?

New example:
image

### CODE REVIEW

File: docs/src/examples/components/Form/Usage/FormExampleCheckbox.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+const theme = {
+  componentStyles: {
+    Checkbox: {
+      root: {
+        padding: 0,
+      },
+    },
+  },
+}
```
Comment (layershifter) [inline docs/src/examples/components/Form/Usage/FormExampleCheckbox.tsx]: Let's remove this one, we definitely should not show this in examples

File: docs/src/examples/components/Form/Usage/FormExampleCheckbox.tsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    name: 'email',
+    id: 'email-inline-shorthand',
+    key: 'email',
+    required: true,
+    inline: true,
+  },
+  {
+    control: {
+      as: Checkbox,
+      labelPosition: 'start',
```
Comment (layershifter) [inline docs/src/examples/components/Form/Usage/FormExampleCheckbox.tsx]:
```suggestion
```

It's not required, so let's omit it

**Comment by layershifter** [comment]
> There are similar examples under Types section. Should I move them under Usage instead?

Yes, please 🐱

File: docs/src/examples/components/Form/Usage/FormExampleCheckbox.tsx (hunk 1 above)
Comment (lucivpav) [reply docs/src/examples/components/Form/Usage/FormExampleCheckbox.tsx]: Without this, however, a padding breaks the visual structure of the form.
image

File: docs/src/examples/components/Form/Usage/FormExampleCheckbox.tsx (hunk 2 above)
Comment (lucivpav) [reply docs/src/examples/components/Form/Usage/FormExampleCheckbox.tsx]: I think it looks better, when the form's structure is unified, i.e. description on the left, user's input on the right.

File: docs/src/examples/components/Form/Usage/FormExampleCheckbox.tsx (hunk 1 above)
Comment (layershifter) [reply docs/src/examples/components/Form/Usage/FormExampleCheckbox.tsx]: Let design team to care about it, we shouldn't tune theme specific things in theme agnostic examples

File: docs/src/examples/components/Form/Usage/FormExampleCheckbox.tsx (hunk 2 above)
Comment (layershifter) [reply docs/src/examples/components/Form/Usage/FormExampleCheckbox.tsx]: We can go whatever you will decide, but should mention:
- that default position is on left across Web:
  - https://www.w3schools.com/tags/tryit.asp?filename=tryhtml5_input_type_checkbox
  - https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox
- we should keep examples minimal and avoid questions like "`labelPosition` is required to use this component"?

File: docs/src/examples/components/Form/Usage/FormExampleCheckbox.tsx (hunk 1 above)
Comment (lucivpav) [reply docs/src/examples/components/Form/Usage/FormExampleCheckbox.tsx]: Did you mean `shouldn't`? 😄

Comment (lucivpav) [reply docs/src/examples/components/Form/Usage/FormExampleCheckbox.tsx]: fixed

**Review by layershifter — APPROVED** [review]

**Pull request merged**